module/zfs/zfs_ioctl.c

```diff
@@ -7018,6 +7052,8 @@ zfs_ioctl_init(void)
 	zfs_ioctl_register_dataset_nolog(ZFS_IOC_TMP_SNAPSHOT,
 	    zfs_ioc_tmp_snapshot, zfs_secpolicy_tmp_snapshot,
 	    POOL_CHECK_SUSPENDED | POOL_CHECK_READONLY);
+	zfs_ioctl_register_pool_modify(ZFS_IOC_POOL_RESCRUB,
```

Comment:
This should probably use the new ioctl signature (like `ZFS_IOC_POOL_INITIALIZE`) rather than the old one. We're trying to avoid adding new legacy ioctls.